artdaq-mu2e/ArtModules/DTCEventVerifier_module.cc

```diff
@@ -140,9 +151,30 @@ bool mu2e::DTCEventVerifier::filter(art::Event& event)
       
 	if ( subEvtWTag != evtWTag) { evtHeader->ewt_check = 0; } //different EWT in a subEvent
 	
+	if (metricMan != nullptr && diagLevel_ > 10)
+	  {
+	    std::ostringstream oss;
+	    oss << "SubEvent: "     << &subEvt - &data.GetSubEvents()[0] 
+		<< ", DTC_ID = "    << dtcID 
+		<< ", EWT_CHECK = " << ( (subEvtWTag != evtWTag) ? 0 : 1)
+		<< ", DTC_CHECK = " << ( ((dtcs_.insert(dtcID).second) && (!isFirstEvent_)) ? 0 : 1);
+	    metricMan->sendMetric("SubEventSnapshot", oss.str(), "xml_string",
+				 metrics_reporting_level_, artdaq::MetricMode::LastPoint);
+	  }
+	
 	//if (subHeader->dtc_mac == dtcHeader->dtc_mac){ evtDTC_ID = dtcID;}
       }//end loop over the subevents
     
+    if (metricMan != nullptr)
+      {
+	std::ostringstream oss;
+	oss << "Fragment: " << &frag - &fragments[0]
+	    << ", EWT_CHECK = " << evtHeader->ewt_check 
+	    << ", DTCs = "      << evtNDTCs << "/" << nDTCs_
```

Comment:
You can send metrics for both evtNDTCs and evtNDTCs/nDTCs_, again, metric collection is very fast and you can send as many as you like.

Reply:
done